### ISSUE TITLE
Add USB ID of Samsung GT-B3710

### ISSUE THREAD

**Issue by mkotsbak** [mkotsbak/Samsung-GT-B3730-linux-driver#23]
Seems it uses the same chip.

**Comment by mkotsbak** [comment]
Hmm, which one is the corresponding IDs for the 3710?:

From usb-ids.gowdy.us/read/UD/04e8 :

6875    GT-B3710 Standalone LTE device (Commercial) 
6876    GT-B3710 LTE Modem  
6877    Galaxy S  
6888    GT-B3730 Composite LTE device (Commercial)  
6889    GT-B3730 Composite LTE device (Commercial)  
689a    LTE Storage Driver [CMC2xx]

**Comment by Dennsen** [comment]
6876 GT-B3710 LTE Modem

**Comment by mkotsbak** [comment]
Dennsen: Japp, I got that from another user too. Have you got it working with the driver?

**Comment by Dennsen** [comment]
No, actually not. Something goes wrong with the usb_modeswitching. I'm not sure if it has to be switched either.
How to best start to investigate it?

**Comment by mkotsbak** [comment]
Actually, it seems that it does not need switching. What does "lsusb -v" tell about the modem? Does it provide both the modem interface and driver disk at the same time?

Have you added the USB ID above to the driver?
